src/main/kotlin/org/rust/lang/core/resolve2/FileModificationTracker.kt

```diff
@@ -77,6 +78,7 @@ private class ModDataLight {
     val enums: MutableList<EnumLight> = mutableListOf()
     val imports: MutableList<ImportLight> = mutableListOf()
     val macroCalls: MutableList<MacroCallLight> = mutableListOf()
+    val procMacroCalls: MutableList<ProcMacroCallLight> = mutableListOf()
```

Comment:
Do I understand correctly that we need this list of proc macro calls to understand if something changed in the call and relaunch name resolution for the corresponding crate?

Reply:
Yes